Source/Soot.cpp

```diff
@@ -0,0 +1,179 @@
+
+
+#ifdef PELEC_USE_SOOT
+#include "PeleC.H"
+#include "SootModel.H"
+#include "SootModel_derive.H"
+#include <Transport.H>
```

Comment:
can you make all the include for headers as quotes instead of carrots? (here and other places)

Reply:
Yeah. Is the general rule to use carrots for libraries like AMReX and quotes for everything else?

Reply:
carrots for amrex and std libs yeah.

Reply:
chevrons :)

Reply:
lol you are right...